lib/rerun/notification.rb

```diff
@@ -34,6 +32,13 @@ def command
         end
       end
 
+      if s.nil? and options[:notify] == true or options[:notify] == "notify-send"
+        if (cmd = command_named('notify-send'))
+          icon_str = "--icon #{icon}" if icon
+          s = "#{cmd} -t 500 --hint=int:transient:1 #{icon_str} \"#{app_name}: #{title} -- #{body}\""
```

Comment:
Rather than putting everything in the summary field, would it make sense to split it across summary and body arguments to notify-send? ie:

```ruby
s = "#{cmd} -t 500 --hint=int:transient:1 #{icon_str} \"#{app_name}: #{title}\" \"#{body}\""
```

At least on the version of GNOME I'm running (3.22.3), long-running summary text ends up getting clipped early because the font is larger, while there's more room for the body (and it expands on mouse-over, unlike the summary).